functions/sendSystemMessage.ts

```diff
@@ -18,54 +17,78 @@ type EnvVars = {
 
 export type Body = {
   taskSid?: string;
+  channelSid?: string;
   message?: string;
   from?: string;
-  newStatus?: string;
   request: { cookies: {}; headers: {} };
 };
 
+export const sendSystemMessage = async (context: Context<EnvVars>, event: Body) => {
+  const { taskSid, channelSid, message, from } = event;
+
+  console.log('------ sendSystemMessage excecution ------');
+
+  if (taskSid && channelSid) {
```

Comment:
Do we really care if they provide both? I'd say a log stating that both were provided so using `channelSid` in preference is the most we need